Library/Homebrew/language/node.rb

```diff
@@ -44,6 +44,9 @@ def self.std_npm_install_args(libexec)
 
       pack = pack_for_installation
 
+      # npm 7 requires that these dirs exist before install
+      FileUtils.mkdir_p "#{libexec}/lib"
```

Comment:
```suggestion
      # npm 7 requires that these dirs exist before install
      (libexec/"lib").mkpath
```

Can you test that locally and ensure it works for you? Thanks!

Reply:
```sb
Failures:

  1) Language::Node#std_npm_install_args does not raise error with a zero exitstatus
     Failure/Error: resp = subject.std_npm_install_args(npm_install_arg)
     
     NoMethodError:
       undefined method `/' for "libexec":String
     # ./language/node.rb:48:in `std_npm_install_args'
     # ./test/language/node_spec.rb:45:in `block (3 levels) in <top (required)>'
     # ./test/spec_helper.rb:195:in `block (3 levels) in <top (required)>'
     # ./test/spec_helper.rb:194:in `block (2 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:124:in `block in run'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `loop'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:110:in `run'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec_ext/rspec_ext.rb:12:in `run_with_retry'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-retry-0.6.2/lib/rspec/retry.rb:37:in `block (2 levels) in setup'
     # ./vendor/bundle/ruby/2.6.0/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `block (2 levels) in <top (required)>'
```

Reply:
https://github.com/Homebrew/brew/blob/c4ae961efac4904aabdd7dc3cbe20dc4ed004869/Library/Homebrew/test/language/node_spec.rb#L28 needs updated to pass `Pathname("libexec")` to match the behaviour of what formulae do.